tests/integration/test_ndim.py

```diff
@@ -34,6 +34,26 @@ def test_python_values(input):
     assert np.ndim(input) == num.ndim(input)
 
 
+def test_ndarray_none():
+    inp = None
+    assert np.ndim(inp) == num.ndim(inp)
+
+
+@pytest.mark.parametrize("input", ([], (), (()), ((), ()), [[]], [[], []]))
+def test_ndarray_empty(input):
+    assert np.ndim(input) == num.ndim(input)
+
+
+@pytest.mark.parametrize("input", ([[0], [1, 2], [3, 4, 5]]))
```

Comment:
Without a trailing comma, the parens just do a superfluous grouping, and this is currently equivalent to:
```
@pytest.mark.parametrize("input",  [[0], [1, 2], [3, 4, 5]])
```
If that is intended I would write as:
```
@pytest.mark.parametrize("input", ([0], [1, 2], [3, 4, 5]))
```
(preferring immutable tuples where reasonable)

Reply:
fix it